sql/src/main/java/io/crate/operation/scalar/arithmetic/AddFunction.java

```diff
@@ -31,9 +32,10 @@
 
 import java.util.List;
 
-public abstract class AddFunction extends ArithmeticFunction {
+public abstract class AddFunction extends ArithmeticFunction implements SymbolFormatter.OperatorFormatter {
 
     public static final String NAME = "add";
+    public static final String OPERATOR = "+";
```

Comment:
add is not an operator, but yeah it written in the same style. at least please rename this static var